### ISSUE TITLE
Hostile NPC`s choose to disengage to attack wildlife

### ISSUE THREAD

**Issue by stk2008** [CleverRaven/Cataclysm-DDA#10342]
Hey all I was messing about and decided to slap an NPC that was in the starting shelter.

Offcourse she became hostile so I went on the attack so did she.

But then all of a sudden she decides to run off to attack a small rat leaving me to be able to kick the hell out of her.

This does not make sense...maybe if it was a zombie but it was a none hostile animal.
##

**Comment by ituluwituluwzev** [comment]
I made a new character in a new world in 6411. Then I teleported him to a forest, and spawned an NPC. The NPC turned hostile and chased my character. During the chase, the NPC ignored several black rats, attacked a fox, and attacked a hostile cougar. When this happened, these animal were between us. Then we ran by two zombie soldiers. While the NPC was closer to zombies than to my character, the zombies were not between us, and the NPC did not disengage the chase.

We can see that hostile NPC only attacks hostile monster that gets in his way.
